ui/src/components/useMessageDialog.tsx

```diff
@@ -111,12 +120,23 @@ const MessageDialogContainer = () => {
     const node = document.createElement("div");
     document.body.appendChild(node);
     messageDialogRef.current = node;
+    register("message", {
+      required: "Please enter a message",
+      minLength: {
+        value: 3,
+        message: "Please enter a message with a minimum of 3 characters",
+      },
+    });
   }, []);
 
-  const postMessage = () => {
+  React.useEffect(() => {
+    if (textareaRef.current) textareaRef.current.value = "";
+  }, [messageDialogParameters.id]);
+
+  const onSubmit: SubmitHandler<FormValues> = (values) => {
     const receiverId = messageDialogParameters.job_poster.id;
     const subject = `${messageDialogParameters.current_user.first_name} ${messageDialogParameters.current_user.last_name} connected with you`;
-    const body = `Message from ${messageDialogParameters.current_user.first_name} ${messageDialogParameters.current_user.last_name}:\n${message}`;
+    const body = `Message from ${messageDialogParameters.current_user.first_name} ${messageDialogParameters.current_user.last_name}:\n${values.message}`;
```

Comment:
I was wondering if this should be done in the backend now that the entire email template is in the backend

Reply:
You mean just passing the raw values and not having to deal with receiverId, subject, body etc? I'm good with that if it's not too much trouble. @subhransu ?

Reply:
Let's leave this as separate future issue.